app/backend/src/couchers/servicers/auth.py

```diff
@@ -214,9 +240,9 @@ def CompleteSignup(self, request, context):
             session.add(user)
             session.commit()
 
-            token = self._create_session(context, session, user)
-
-            return auth_pb2.AuthRes(token=token, jailed=user.is_jailed)
+            token, expiry = self._create_session(context, session, user, False)
+            context.send_initial_metadata((("set-cookie", create_session_cookie(token, expiry)),))
```

Comment:
It might be easier to read as `context.send_initial_metadata([("set-cookie", create_session_cookie(token, expiry))])`. Or not, feel free to ignore :)